### PR TITLE
Quick links element

### PR DESCRIPTION
**Pull request by amitaibu**
| Mobile | Tablet | Desktop | RTL (showing hover)| Node view | Node edit |
| -- | -- | -- | -- | -- | -- |
| image | image | image | image | image | image |

### CODE REVIEW

**Comment by amitaibu** [comment]
Added migrate

image

**Comment by amitaibu** [comment]
@isapisa @nora-novak fyi ^^

File: web/themes/custom/server_theme/server_theme.theme [hunk 1]
```diff
@@ -443,11 +443,12 @@ function server_theme_theme() {
     ],
   ];
 
-  // A single icon.
-  $info['server_theme_icon'] = [
```
Comment (mariano-dagostino) [inline web/themes/custom/server_theme/server_theme.theme:447]: Is this correct?

File: web/themes/custom/server_theme/templates/server-theme-card--quick-link-item.html.twig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% import '@server_theme/templates/server-theme-card-layout-base.html.twig' as base %}
+
+<a href="{{ url }}" class="{{ base.getBaseClasses() }} group flex flex-col justify-between gap-y-3 md:gap-y-5 p-6 overflow-hidden bg-gray-200 hover:bg-gray-700">
+
+    <div class="group-hover:text-white">
+      {{ items }}
+    </div>
+
+    <div class="h-12 w-12 bg-gray-50 group-hover:bg-gray-200 rounded-full grid place-content-center rtl-flip">
+      {# Heroicons: arrow-right-circle #}
```
Comment (mariano-dagostino) [inline web/themes/custom/server_theme/templates/server-theme-card--quick-link-item.html.twig:10]: Should this be a button instead?

**Review by mariano-dagostino — APPROVED** [review]
@amitaibu Looks good. A few comments.

File: web/themes/custom/server_theme/server_theme.theme (hunk 1 above)
Comment (amitaibu) [reply web/themes/custom/server_theme/server_theme.theme:447]: Yes, it was a wrong copy past from older PR

File: web/themes/custom/server_theme/templates/server-theme-card--quick-link-item.html.twig (hunk 2 above)
Comment (amitaibu) [reply web/themes/custom/server_theme/templates/server-theme-card--quick-link-item.html.twig:10]: The entire card is inside an `a href` so I think this doesn't need to be a button. It's just for decoration purposes.

File: web/modules/custom/server_general/src/Plugin/EntityViewBuilder/ParagraphQuickLinks.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+   */
+  public function buildFull(array $build, ParagraphInterface $entity): array {
+    /** @var \Drupal\Core\Field\EntityReferenceFieldItemListInterface $paragraphs */
+    $paragraphs = $entity->get('field_quick_link_items');
+    $items = $this->buildReferencedEntities($paragraphs, 'full', $entity->language()->getId());
+
+    if (empty($items)) {
+      // While building the quick link items, we checked and got that user has
+      // no access to any item.
+      return [];
```
[the remark this answers is not in the document]
Comment (amitaibu) [reply web/modules/custom/server_general/src/Plugin/EntityViewBuilder/ParagraphQuickLinks.php]: Done

**Pull request merged**